### ISSUE TITLE
Right click menu is buggy

### ISSUE THREAD

**Issue by tarakciemre** [open-source-labs/Svelvet#499]
When right clicked, the right-click menu is displayed in a buggy manner and you cannot change properties of nodes. This is true for right-clicking on a component on the official svelvet website demonstration as well

**Comment by devnull03** [comment]
yeah I also have the same issue, there is no way to disable the right-click menu it seems like either

**Comment by devnull03** [comment]
Okay i just figured out, the right-click menu is just another Node and you can use the `editable={false}` attribute on a Node to disable the menu, wrap it with a div to detect a right-click event and create your own.
